nri-kafka/test/Main.hs

```diff
@@ -2,11 +2,15 @@ module Main (main) where
 
 import qualified Spec.Kafka.Worker.Integration
 import qualified Spec.Kafka.Worker.Partition
+import qualified System.Environment
 import qualified Test
 import qualified Prelude
 
 main :: Prelude.IO ()
-main = Test.run tests
+main = do
+  -- macos runners seem to be slow and fail on several kafka integration tests
+  System.Environment.setEnv "NRI_TEST_TIMEOUT" "240000"
```

Comment:
Wow this is a lot.
I thought it was just a temporary thing while you were debugging but I see it's still there.
I guess if we need it, we need it!
How long do the tests actually take?